pages/code-of-conduct.tsx

```diff
@@ -10,89 +10,160 @@ export default withPageMetadata((props: WithPageMetadataProps) => (
   >
     <h1>Code of Conduct</h1>
 
+    <h2>Purpose</h2>
     <p>
-      All attendees, speakers, sponsors and volunteers at our conference are required to agree with the following code
-      of conduct. We are expecting cooperation from all participants to help ensuring a safe, welcoming and inclusive
-      environment for everybody.
+      {props.pageMetadata.conference.Name} is dedicated to providing a harassment-free conference experience for
+      everyone, regardless of gender, gender identity and expression, sexual orientation, disability, physical
+      appearance, body size, race, age or religion. We do not tolerate harassment of conference participants in any
+      form. Sexual language and imagery is not appropriate for any conference venue, including talks.
```

Comment:
The "for any conference venue" bit reads weirdly?

Reply:
"Sexual language and imagery is not appropriate anywhere. This applies to talks also.

Reply:
"regardless of but not limited to" ? Not sure it's really required though.

Reply:
Yes, agree. I've gone with Rebecca's suggestion "Sexual language and imagery is not appropriate anywhere. This applies to talks also."